### DISCUSSION TITLE
Unmasking Elements

### DISCUSSION THREAD

**Discussion by hanan-commits-work** [iTwin/itwinjs-core#3823] [Q&A]
I want to unmask previously masked elements. I tried doing 
maskModel.planarClipMaskSettings = undefined

and the realityModel gets unmasked, but I cannot do any element selection anymore.
Kindly guide for a better approach.

**Comment by pmconne** [comment]
The more of your code you show, the more able we will be to help you.

**Reply by hanan-commits-work** [reply]
I have made a workaround on that. the problem was that the classifiers were not set in the original masking.
But I want to know what is the recommended approach on removing the mask.

I am simply making the planarClipMaskSettings = undefined.
Is this approach recommended or is there a better way to do this without reattaching RealityModels.

**Comment by pmconne** [comment]
Setting planarClipMaskSettings to undefined should be fine.
If a problem remains please describe it clearly and show relevant code.
